docs/user_guide/bank/getting_started_customization.md

```diff
@@ -76,6 +81,9 @@ If you do not pick a day, no reminder emails are sent.
 #### Deadline day (Final day of the month to submit Requests)
 This day will be included in the reminder email message,
 
+#### Additional text for reminder email
+If present, this text will be included in the reminder email placed at the end of the email. In the above email template, this additional message is placed in the position of `<%= @organization.reminder_email_text %>`.
+
```

Comment:
If present, this text will be included in the reminder email after the default text. In the above email template, this additional message will replace [Additional text for reminder email (see below)].